### ISSUE TITLE
Create In-Place Variants for new Wedges

### ISSUE THREAD

**Issue by GeorgeR227** [AlgebraicJulia/Decapodes.jl#208]
PR #201 introduced new wedge operators for various dual cases but as of now these are all out-of-place operators. Mirroring the pp wedges structure, we should include wrappers to provide an in-place and an out-of-place version for dec_matrix_generate and update gensim to unlock this functionality for simulation generation.

**Comment by lukem12345** [comment]
Good catch, George. Could you make a quick set of instructions for how to add such support?

**Comment by GeorgeR227** [comment]
1. Create a wrapper that generates both the in-place and out-of-place functions and returns them as a tuple in that order. See https://github.com/AlgebraicJulia/Decapodes.jl/blob/393e3279e6d8b385dcf07381347578a0de476050/src/operators.jl#L97 as an example, your actual implementation may differ based what the function looks like.
2. Add the symbols for those wedge products into the set extra_dec_operators here https://github.com/AlgebraicJulia/Decapodes.jl/blob/393e3279e6d8b385dcf07381347578a0de476050/src/simulation.jl#L513 This is just a collection of non-matrix but in-place operators.

That should be it but it's been a while since I've done this and I remember the solution to getting the wedge to work was kinda hacky. Lmk if anything goes awry.

**Comment by lukem12345** [comment]
We will hold off on this feature since it would be best to introduce the in-place version of these operators at the same time we add GPU support for these operators. GPU kernels are always in-place.

**Comment by lukem12345** [comment]
This issue can be addressed by the current push in CombinatorialSpaces towards kernelizing operators https://github.com/AlgebraicJulia/CombinatorialSpaces.jl/issues/113 In-place operators and their GPU versions are now one-and-the-same.
